README.md

```diff
@@ -1,15 +1,13 @@
 # Fulfil.io Rubygem
 
-[![CircleCI](https://circleci.com/gh/knowndecimal/fulfil.svg?style=svg&circle-token=da80ea6500af15b3a795a3913efe35742bab94c1)](https://circleci.com/gh/knowndecimal/fulfil)
```

Comment:
GH has a badge!

`![example workflow](https://github.com/<OWNER>/<REPOSITORY>/actions/workflows/<WORKFLOW_FILE>/badge.svg)`

Reply:
I'll add that one!